.vs/HotelApp/config/applicationhost.config

```diff
@@ -157,7 +157,7 @@
       </site>
       <site name="HotelApp.API" id="2">
```

Comment:
gdje ti se točno nalazi .gitignore file? Ovaj .vs folder nebi smio završit na sourceu